specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2018-02-01-preview/examples/BuildStepsGet.json

```diff
@@ -0,0 +1,44 @@
+{
+  "parameters": {
+    "subscriptionId": "4385cf00-2d3a-425a-832f-f4285b1c9dce",
+    "resourceGroupName": "myResourceGroup",
+    "registryName": "myRegistry",
+    "api-version": "2018-02-01-preview",
+    "buildDefinitionName": "myBuildDefinition",
+    "stepName": "dockerbuild"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "type": "Docker",
+        "name": "dockerbuild",
+        "branch": "master",
+        "imageName": "azurerest:testtag",
+        "dockerFilePath": "DockerFile",
+        "isPushEnabled": true,
+        "buildArguments": [
+          {
+            "type": "DockerBuildArgument",
+            "name": "mytestargument",
+            "value": "mytestvalue",
+            "isSecret": false
+          },
+          {
+            "type": "DockerBuildArgument",
+            "name": "mysecrettestargument",
+            "value": "mysecrettestvalue",
+            "isSecret": true
+          }
+        ],
+        "baseImageDependencies": [
+          {
+            "imageName": "microsoft/aspnetcore",
+            "tag": "2.0",
```

Comment:
Per docker's terminology, image name would be `repository:tag` so here
`microsoft/aspnetcore` is repository, `2.0` is tag, `microsoft/aspnetcore:2.0` is image name.

Reply:
So you suggest to rename it to repository name?